src/utils/num.ts

```diff
@@ -97,28 +168,55 @@ export function assertInRange(
 
 /**
  * Convert BigNumberish array to decimal string array
+ * @param rawCalldata Array of BigNumberish values
  * @returns format: decimal string array
+ * @example
+ * ```typescript
+ * const bigNumberishArray: BigNumberish[] = [123, "456", 789n];
+ * const result = bigNumberishArrayToDecimalStringArray(bigNumberishArray);
+ * // result = ["123", "456", "789"]
+ * ```
  */
 export function bigNumberishArrayToDecimalStringArray(rawCalldata: BigNumberish[]): string[] {
   return rawCalldata.map((x) => toBigInt(x).toString(10));
 }
 
 /**
  * Convert BigNumberish array to hexadecimal string array
+ * @param rawCalldata Array of BigNumberish values
  * @returns format: hex-string array
+ * @example
+ * ```typescript
+ * const bigNumberishArray: BigNumberish[] = [123, "456", 789n];
+ * const result = bigNumberishArrayToHexadecimalStringArray(bigNumberishArray);
+ * // result = ["0x7b", "0x1c8", "0x315"]
+ * ```
  */
 export function bigNumberishArrayToHexadecimalStringArray(rawCalldata: BigNumberish[]): string[] {
   return rawCalldata.map((x) => toHex(x));
 }
 
 /**
  * Test if string is whole number (0, 1, 2, 3...)
+ * @example
+ * ```typescript
+ * const result = isStringWholeNumber("123");
+ * // result = true
+ * ```
  */
 export const isStringWholeNumber = (value: string) => /^\d+$/.test(value);
 
 /**
  * Convert string to decimal string
  * @returns format: decimal string
+ * @example
+ * ```typescript
+ * const result = getDecimalString("0x1a");
+ * // result = "26"
+ * 
+ * const result2 = getDecimalString("Hello");
+ * // result2 = "Hello need to be hex-string or whole-number-string"
```

Comment:
inform that it's an error, with `Throws Error:`

Reply:
change from "// result2 = 'Hello need to be hex-string or whole-number-string' "  to 
Throw Error: "Hello need to be hex-string or whole-number-string"